### PR TITLE
Specified cookie domain bugfix

### PR DESCRIPTION
**Pull request by DShumkov**
I've found a bug when I use Session::set_cookie_domain('.example.com'). I think methods: open() and destroy() at HybridSessionStore_Cookie don't work normally cause domain name parmeter missed on Cookie::set and Cookie::force_expiry in case we use specific cookie domain.

### CODE REVIEW

**Comment by DShumkov** [comment]
Hmmmm 
I've run tests on my local env - it is all ok.

File: code/HybridSessionStore.php [hunk 1]
```diff
@@ -273,7 +273,16 @@ public function open($save_path, $name) {
...
-		if ($this->currentCookieData) Cookie::set($this->cookie, '');
+		$params = session_get_cookie_params();
+		if ($this->currentCookieData) {
+			Cookie::set(
+			$this->cookie, 
+			'', 
+			90, 
+			$params['path'], 
+			$params['domain'], 
+			$params['secure']);
```
Comment (tractorcow) [inline code/HybridSessionStore.php]: Should you set `$params['httpOnly']` here too?

Comment (DShumkov) [inline code/HybridSessionStore.php]: Yeah, my bad.

**Comment by tractorcow** [comment]
Do you know why travis is failing with a parse error?

**Comment by tractorcow** [comment]
`PHP Parse error:  syntax error, unexpected T_VARIABLE in /home/travis/builds/ss/hybridsessions/code/HybridSessionStore.php on line 245`

**Comment by tractorcow** [comment]
:D

**Comment by dhensby** [comment]
we still have test failures

**Comment by wilr** [comment]
Fixed in the #27

**Pull request closed without merging**